### PR TITLE
Feature 22 override bootstrap variables

### PR DESCRIPTION
**Pull request by alvincases**

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -26,21 +25,23 @@
...
-                    <li><a href="#">sponsors</a></li>
-                    <li><a href="#">pricing</a></li><br><br>
-                  </ul>
-                <div class="register-button col-12"><a href="#">register</a></div>
+                <ul class="list-group" col-12 col-md-12">
+                  <li class="list-group-item"><a href="#">Venue</a></li>
+                  <li class="list-group-item"><a href="#">Sponsors</a></li>
+                  <li class="list-group-item"><a href="#">Pricing</a></li>
+                </ul>
+                <button type="button" class="btn btn-primary">Register</button>
```
Comment (Luukyb) [inline static/index.html]: This needs to be a link I think. Where would we place the link to /register?

Comment (marychris) [reply static/index.html]: @alvincases , please wrap 'Register' in <a></a> tag.

File: static/index.html [hunk 2]
```diff
@@ -60,25 +61,25 @@ <h1>Conference about Open Source</h1>
...
         <div class="container">
           <div class="row">
-            <div class = "sponsor-region col-lg-12">
+            <div class="sponsor-region col-lg-12">
               <h2>Sponsors</h2>
               <p>Your contribution will help lower ticket prices for attendees, and as a sponsor, there is something in it for you</p>
-              <div class = "link-btn">  
-                <a href="#">Sponsor now</a>
+              <div class="sponsor-btn">
+                <button type="button" class="btn btn-outline-primary">Sponsor Now</button>
```
Comment (marychris) [inline static/index.html]: @alvincases ,please wrap 'Sponsor now' in <a></a> tag.

File: static/index.html [hunk 3]
```diff
@@ -60,25 +61,25 @@ <h1>Conference about Open Source</h1>
...
               </div>
               <h4>Gold Sponsors</h4>
-              <div class ="sponsors-image">
+              <div class="sponsors-image">
                 <img src="images/Promet_Logo_Lockup.png"  alt="Promet Source logo.">
```
Comment (marychris) [inline static/index.html]: @alvincases please wrap img tag inside <a> tag and input href="https://www.prometsource.com/".
<a href="https://www.prometsource.com/"></a>

File: static/sass/partials/_override-variables.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// Override variables.
+
+// Buttons.
+$btn-padding-y:13px;
```
Comment (Luukyb) [inline static/sass/partials/_override-variables.scss]: Missing space.

File: static/sass/partials/_override-variables.scss [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+$btn-padding-x:13px;
```
Comment (Luukyb) [inline static/sass/partials/_override-variables.scss]: Missing space.

File: static/sass/partials/_override-variables.scss (hunk 4 above)
Comment (alvincases) [reply static/sass/partials/_override-variables.scss]: Done

File: static/sass/partials/_override-variables.scss (hunk 5 above)
Comment (alvincases) [reply static/sass/partials/_override-variables.scss]: Done

**Pull request merged**